less/transferwise.less

```diff
@@ -56,7 +56,7 @@ a.btn {
 }
 
 .nav > li > a {
-    padding: 11px 16px 9px;
+    padding: 11px 16px 5px;
```

Comment:
ok, I checked this.  I've actually removed this rule from that file in my latest set of local changes, however, there is a corresponding rule in navs.less, with padding: 10px 16px;  We need to keep the spacing even either side, I'd suggest changing the version in navs.less to 8px 16px and making a rule to reposition the glyphicon in that context e.g ... > li > a > .glyphicon { ... }  Ok to make that change?